modules/images-configuration-parameters.adoc

```diff
@@ -80,3 +80,8 @@ registries are blocked.
 Only one of `blockedRegistries` or `allowedRegistries` may be set
 
 |===
+
+[WARNING]
+====
+When the `allowedRegistries` parameter is defined, _all_ registries including `registry.redhat.io` and `quay.io` are blocked unless explicitly listed. If using the parameter, declare all registries required within your environment to prevent pod failure.
```

Comment:
s/_all_/all
s/pod/Pod per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
Thank you Ashley! Changes now pushed to this PR.